tests/performance_tests/log_reader.py

```diff
@@ -360,19 +360,14 @@ def createReport(guide: chainBlocksGuide, targetTps: int, testDurationSec: int,
     report['nodeosVersion'] = Utils.getNodeosVersion()
     return report
 
-def createJSONReport(guide: chainBlocksGuide, targetTps: int, testDurationSec: int, tpsLimitPerGenerator: int, tpsStats: stats, blockSizeStats: stats,
-                     trxLatencyStats: basicStats, trxCpuStats: basicStats, trxNetStats: basicStats, testStart: datetime, testFinish: datetime, argsDict, completedRun) -> json:
-    report = createReport(guide=guide, targetTps=targetTps, testDurationSec=testDurationSec, tpsLimitPerGenerator=tpsLimitPerGenerator, tpsStats=tpsStats, blockSizeStats=blockSizeStats,
-                     trxLatencyStats=trxLatencyStats, trxCpuStats=trxCpuStats, trxNetStats=trxNetStats, testStart=testStart, testFinish=testFinish, argsDict=argsDict, completedRun=completedRun)
-    return reportAsJSON(report)
-
 def reportAsJSON(report: dict) -> json:
     report['testStart'] = "Unknown" if report['testStart'] is None else report['testStart'].isoformat()
     report['testFinish'] = "Unknown" if report['testFinish'] is None else report['testFinish'].isoformat()
     return json.dumps(report, sort_keys=True, indent=2)
 
 def calcAndReport(data, targetTps, testDurationSec, tpsLimitPerGenerator, nodeosLogPath, trxGenLogDirPath, blockTrxDataPath, blockDataPath,
```

Comment:
Have things stabilized enough to consolidate some of these arguments into classes to cut down on how verbose the method signature is?  I could see bundling all the paths, especially.

Reply:
I'll make a follow-on ticket to address this.